src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2717,18 +2717,18 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 						// Inline the image if no preview image will be generated (see #636)
 						if ($objFile->height !== null && $objFile->height <= 50 && $objFile->width !== null && $objFile->width <= 400)
 						{
-							$thumbnail .= '<br><img src="' . $objFile->dataUri . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="" style="margin:0 0 2px -18px">';
+							$thumbnail .= '<br><img src="' . $objFile->dataUri . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="" style="margin:0 0 2px -18px;max-width:400px;max-height:50px;width:auto;height:auto">';
```

Comment:
This change seems redundant. The if condition already ensures that the image is not bigger than 400x50.

Reply:
True, I have removed that style now.